src/test.rs

```diff
@@ -35,7 +35,6 @@
 //! # use warp::Filter;
 //! #[tokio::test]
 //! async fn test_sum() {
-//! #    let sum = || warp::any().map(|| 3);
```

Comment:
Isn't removing this going to cause the next line to fail?

Reply:
hey @seanmonstar, thank you for your comment. I'm new to Rust. Would you help me understand the `#`-prefixed lines in the example code? In the meantime, I reverted the change.

Reply:
This is something specific to Rust documentation. It will "hide" the line from the rendered documentation. It's useful to setting up things to make the compiler happy, but would other just be a distraction from the example.

So in this case, the rest of the code references a `sum` filter, but I didn't want to write out a full sum example here (users can go find it in the routing docs), so I just hid the "fake" one.

Reply:
Awesome! Thank you so much the for the explanation 🙏.. Pretty neat! Does the PR look good to you otherwise? Have a great Friday afternoon!